desktop/cmp/input/Select.js

```diff
@@ -145,12 +145,25 @@ export class Select extends HoistInput {
          */
         queryBuffer: PT.number,
 
+
+        /**
+         * Function to be used for filtering of values for a given query string  input.
+         * Used for filtering of options provided by `options` prop when `enableFilter` is true.
+         * Not to be confused with `queryFn` prop, used in asynchronous mode.
+         *
+         * Provided function should take an option and a query value and return a boolean.
+         * Defaults to a case-insensitive match on word starts.
+         */
+        filterOption: PT.func,
```

Comment:
I know this is the name used by React-Select, but I find the name of this prop unintuitive. Could we use `filterFn` to be consistent with `queryFn`?

Reply:
Makes sense, since we long ago chose `queryFn` rather than the underlying `loadOptions` property.